diff --git a/scripts/solid-plugin.ts b/Users/mayfield/.bun/install/cache/@opentui/solid@0.1.88@@@1/scripts/solid-plugin.ts
deleted file mode 100644
index 78f4c937563c087510a05da4f6ad1a1693f67703..0000000000000000000000000000000000000000
diff --git a/src/types/elements.d.ts b/src/types/elements.d.ts
index 5d228e2356f0106b3d86301636a0dec95add2841..991abf65d812cb8e1b9b66ef5ea0fec11095057f 100644
--- a/src/types/elements.d.ts
+++ b/src/types/elements.d.ts
@@ -30,12 +30,17 @@ export type TextProps = ComponentProps<TextOptions, TextRenderable> & {
 };
 export type SpanProps = ComponentProps<{}, TextNodeRenderable> & {
     children?: TextChildren | Array<TextChildren>;
+    color?: string;
+    bold?: boolean;
+    italic?: boolean;
+    underline?: boolean;
 };
 export type LinkProps = SpanProps & {
     href: string;
 };
 export type BoxProps = ComponentProps<ContainerProps<BoxOptions>, BoxRenderable> & {
     focused?: boolean;
+    background?: string;
 };
 export type InputProps = ComponentProps<InputRenderableOptions, InputRenderable> & {
     focused?: boolean;
